### PR TITLE
chore!: remove query field on custom emojis listing

### PR DESCRIPTION
**Pull request by ricardogarim**

### CODE REVIEW

**Review by MarcosSpessatto — APPROVED** [review]

**Review by gabriellsh — CHANGES_REQUESTED** [review]
Although this is a chore, the behaviour of the endpoint has slightly changes (common used fields passed through `query` param are now regular params)

So this could be a feat, but even if it isn't, this change requires a changeset. Can you add the changeset please, and evaluate if it is a feat or improve?

File: .changeset/perfect-wolves-impress.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@rocket.chat/rest-typings': patch
+'@rocket.chat/meteor': patch
```
Comment (MarcosSpessatto) [inline .changeset/perfect-wolves-impress.md:3]:
```suggestion
'@rocket.chat/rest-typings': major
'@rocket.chat/meteor': major
```

**Review by MarcosSpessatto — DISMISSED** [review]

**Review by gabriellsh — APPROVED** [review]

File: .changeset/perfect-wolves-impress.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+---
+
+Changed custom emoji listing endpoint by moving query params from the 'query' attribute to standard query parameters.
```
Comment (gabriellsh) [inline .changeset/perfect-wolves-impress.md:6]:
```suggestion
Changes custom emoji listing endpoint by moving query params from the 'query' attribute to standard query parameters.
```

Comment (gabriellsh) [reply .changeset/perfect-wolves-impress.md:6]: Please adhere to the changeset standard.

**Review by gabriellsh — CHANGES_REQUESTED** [review]

File: packages/rest-typings/src/v1/emojiCustom.ts [hunk 3]
```diff
@@ -40,8 +37,16 @@ const emojiCustomListSchema = {
 			type: 'string',
 			nullable: true,
 		},
+		_updatedAt: {
+			type: 'string',
+			nullable: true,
```
Comment (KevLehman) [inline packages/rest-typings/src/v1/emojiCustom.ts:42]: No need for nullable

**Review by KevLehman — CHANGES_REQUESTED** [review]

**Review by MarcosSpessatto — APPROVED** [review]

**Review by KevLehman — APPROVED** [review]

**Pull request merged**